sdk/python/core/docsgen/delete_operation.rst

```diff
@@ -0,0 +1,55 @@
+Delete operation
+****************
+This document explains how to use YDK object to delete nodes. Examples below use `ydktest.json <https://raw.githubusercontent.com/CiscoDevNet/ydk-gen/master/profiles/test/ydktest.json>`_ profile file to generate YDK test package.
+
+Let's write some boilerplate code for device connection:
+
+.. code-block:: python
+
+    from ydk.services import CRUDService
+    from ydk.providers import NetconfServiceProvider
+    from ydk.models import ydktest_filterread as ysanity
+    ncc = NetconfServiceProvider(address='127.0.0.1',
+                                 username='admin',
+                                 password='admin',
+                                 protocol='ssh',
+                                 port=12022)
+    crud = CRUDService()
+
+Delete operation could be executed on YANG container, list and items in  YANG list or leaf-list, to delete a container:
```

Comment:
How about something like:

"The delete operation can be executed on YANG containers and lists. Specific items in YANG list or leaf-list can also be deleted. To delete a container:"